beater/api/mux.go

```diff
@@ -69,6 +72,13 @@ const (
 	IntakeRUMPath = "/intake/v2/rum/events"
 
 	IntakeRUMV3Path = "/intake/v3/rum/events"
+
+	// OtlpTracesIntakePath defines the path to ingest OpenTelemetry traces (HTTP Collector)
```

Comment:
```suggestion
	// OTLPTracesIntakePath defines the path to ingest OpenTelemetry traces (HTTP Collector)
```

etc.

See https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Refactored initialisms in 134858f79. Thank you for the link !